.editorconfig

```diff
@@ -0,0 +1,366 @@
+root = true
+
+# All files
+[*]
+indent_style = space
+
+# Xml files
+[*.xml]
+indent_size = 2
+
+# C# files
```

Comment:
This repository will contain JavaScript code, for that reason I think it should delete the csharp references and have configurations for JS.

Reply:
@brogdonm Like so?

Reply:
👍